<launch>
  <node name="radar_node_k79" pkg="radar_ros_interface" type="radar_node_k79" args="--host-ip 10.0.0.77 --host-port 1024 --radar-ip 10.0.0.10 --radar-port 1024" output="screen" required="true" />
</launch>
